src/language/class-modifiers.md

```diff
@@ -0,0 +1,245 @@
+---
+title: Class modifiers
+description: Modifier keywords for class declarations to control external library access.
+---
+
+Class modifiers control how a class or a mixin can be subtyped in libraries
+outside of the library where it's defined.
+
+The full set of modifiers that can appear before a class declaration
+are `abstract`, [`mixin`][mixin], `base`, `final`, `interface`, and `sealed`. 
+Only the `base` modifier can appear before a mixin declaration. The modifiers do
+not apply to other declarations like `enum`, `typedef`, or `extension`.
+
+When deciding whether to use class modifiers, consider the intended uses of the
+class, and what behaviors the class needs to be able to rely on.
+
+## No modifier
+
+Use a `class` or `mixin` declaration without a modifier to allow unrestricted
+permission to create subtype relationships from outside libraries.
```

Comment:
To me, it's a little odd to say "from outside libraries" here, since `abstract` doesn't place restriction based on the library, it applies to all code that tries to subclass it.

```suggestion
permission to create subtype relationships from any library.
```

Reply:
Yeah... maybe moving that here was a mistake because it makes it harder to summarize the whole page. I'll fix it, but need to make sure that "outside libraries" is explicitly stated for all of the ones that it does affect